WcaOnRails/config/locales/en.yml

```diff
@@ -925,11 +936,23 @@ en:
         one: "WCA Delegate"
         other: "WCA Delegates"
       contact: "Contact"
-      competitor_limit: "Competitor Limit"
-      entry_fee: "Entry Fee"
       information: "Information"
       events: "Events"
       registration_requirements: "Registration requirements"
+      create_wca_account_html: "Create a WCA account %{here} if you don't have one."
+      claim_wca_id_html: "If this is not your first competition associate your WCA ID to your WCA account %{here}."
+      register_html: "Register for this competition %{here}."
+      competitor_limit: "There is a competitor limit of %{competitor_limit} competitors."
+      no_competitor_limit: "There isn't a competitor limit."
```

Comment:
nit: I think "There is no competitor limit" reads more clearly here.

Reply:
Ok!